src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -19,7 +19,7 @@ static Https_ClientCredentialTypeTests()
         s_password = "wcfSaysHell0World!";
     }
 
-    [ConditionalFact(nameof(Root_Certificate_Installed))]
+    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
```

Comment:
Reviewers: I added these new conditions only to tests known failing in CI.  This allows them to run in self-host today.  After @hondai investigates what web-site configuration pivots we need, we'll make the detectors for these real.